src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String getDrawable();
```

Comment:
```suggestion
    void draw();
```

<img width="973" alt="image" src="https://user-images.githubusercontent.com/68428976/169119503-2c4a2e72-3a6d-4dad-a4b7-8999aedc5529.png">